source/libs/on-new-comments.js

```diff
@@ -0,0 +1,35 @@
+import once from 'onetime';
```

Comment:
This file is not currently used for this feature, but it's still valid for generic  comment features except this one.